components/sysutils/cronbase/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Adam Stevko 
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		cronbase
+COMPONENT_VERSION=	1.0
+COMPONENT_SUMMARY=	cronbase holds root crontab and cron directories.	
+COMPONENT_PROJECT_URL=	http://www.openindiana.org
+COMPONENT_FMRI=		system/cron/base
```

Comment:
Is the idea that this package delivers the config files?  How will this eventually play with mediators and the illumos-gate cron bits?